cmd/machine-controller/main.go

```diff
@@ -37,11 +36,16 @@ func main() {
 	logs.InitLogs()
 	defer logs.FlushLogs()
 
-	driver := cp.NewAzureDriver(&spi.PluginSPIImpl{})
-
+	driver := provider.NewDefaultDriver(access.NewDefaultAccessFactory())
 	if err := app.Run(s, driver); err != nil {
-		fmt.Fprintf(os.Stderr, "%v\n", err)
```

Comment:
startup error not being logged after port

Reply:
addressed in [commit](https://github.com/gardener/machine-controller-manager-provider-azure/pull/105/commits/0eff475780d32d7d94991421ada3e6dd3f01710d)